modules/nf-core/rgi/main/main.nf

```diff
@@ -24,7 +27,53 @@ process RGI_MAIN {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def db_download_cmd = ""
+    def db_card_cmd = ""
+    def db_wildcard_cmd = ""
+    def load_wildcard = ""
+    def copy_db_cmd = "cp $database/card* database_output"
+
+    if (wildcard.equalsIgnoreCase("yes")) {
+        if (!database) { // Download both DBs if no path is given
+            db_download_cmd = "wget https://card.mcmaster.ca/latest/data;tar -xvf data ./card.json -C database_output;wget -O wildcard_data.tar.bz2 https://card.mcmaster.ca/latest/variants;mkdir -p wildcard;tar -xjf wildcard_data.tar.bz2 -C wildcard;gunzip wildcard/*.gz"
+            db_card_cmd = "rgi card_annotation -i database_output/card.json > card_annotation.log 2>&"
+            db_wildcard_cmd = "rgi wildcard_annotation -i wildcard --card_json database_output/card.json -v \$DB_VERSION > wildcard_annotation.log 2>&1"
```

Comment:
`rgi card_annotation` etc should be separate modules!

Reply:
The separate module `rgi/cardannotation` is in this PR now! I figured it makes more sense than a new PR because it is an optional dependency of `rgi/main` and we can test here that everything works together as expected. Which it does now! :raised_hands: